src/test/java/com/rocana/lucene/codec/v1/TestRocanaSearchCodecV1.java

```diff
@@ -32,10 +33,53 @@
 
 /**
  * Unit test for {@link RocanaSearchCodecV1}.
+ *
+ * This test is not a fork of Lucene code.
```

Comment:
Good comment. Possible enhancement: "this test class provides additional Rocana codec specific tests in addition to the tests forked from Lucene".  I had to think for a second what the terse comment above meant.